.changeset/grumpy-files-heal.md

```diff
@@ -0,0 +1,11 @@
+---
+"@latticexyz/cli": major
+"@latticexyz/protocol-parser": minor
+"@latticexyz/store": major
+"@latticexyz/world": major
+---
+
+- Add `FieldLayout`, which is similar to `Schema`, but with static byte lengths instead of `SchemaType`s
```

Comment:
Since this is a major change, can we add more info here? Explain what is/was `Schema`, what is `FieldLayout`, and why we're making this change?

Reply:
added some more info, not sure how far to go, do tell if u want even more details